src/components/Avatar/Avatar.tsx

```diff
@@ -101,30 +101,35 @@ class Avatar extends UIComponent<Extendable<IAvatarProps>, any> {
   }
 
   renderComponent({ ElementType, classes, rest, styles, variables }) {
-    const { src, alt, name, status, getInitials, size } = this.props as IAvatarPropsWithDefaults
+    const { name, status, image, label, getInitials, size } = this.props as IAvatarPropsWithDefaults
 
     return (
       <ElementType {...rest} className={classes.root}>
-        {src ? (
-          <Image styles={styles.imageAvatar} fluid avatar src={src} alt={alt} title={name} />
-        ) : (
-          <Label
-            styles={styles.avatarNameContainer}
-            as="div"
-            content={getInitials(name || '')}
-            variables={{ padding: '0px' }}
-            circular
-            title={name}
-          />
-        )}
+        {Image.create(image, {
+          defaultProps: {
+            fluid: true,
+            avatar: true,
+            title: name,
+            styles: styles.image,
+            variables: variables.image,
+          },
+        })}
+        {!image &&
+          Label.create(label || {}, {
```

Comment:
currently it is possible to use component in the following way:

```jsx
<Avatar name='John Doe' label={{ content: 'A' }} />
```

In that case the `name`'s value will be completely ignored. Could we consider `name` to be a shorthand slot (with `Label` being a default component)?

Reply:
The name and label are two different things. The name is used as a title for the Label and Image components inside the Avatar, but the content of the Label is actually the result of getInitials method (prop), that uses the names for generating the initials. The use can however override the content of the Label using the shorthand, and the name can still reflects the actual user's name.

Reply:
frankly, a bit unsure whether we do need both `label` and `name` at the same time - it seems that for almost all cases it would be sufficient to have just one prop. Let's please, discuss that tomorrow